Kernel/Ordering_Equality.cpp

```diff
@@ -45,21 +45,11 @@ class Ordering::EqCmp
   }
 
   Result compare_s1Gt1(TermList s1,TermList s2,TermList t1,TermList t2) const;
```

Comment:
Maybe this entire class could be just `static` methods, I think having an object of this class lost its value some refactors before. Having class members `s1,s2,t1,t2` is especially confusing since then we use arguments with the same name in almost all methods. What do you think?

Reply:
Yes, please! 

The `s1,s2,t1,t2;` members can become local in `compareEqualities` and that's it, right?

Reply:
Yep, that should be it.